.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,29 @@
+name: Run Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+
+  release:
+    types:
+      - created
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '14'
```

Comment:
let's use node 18

Reply:
actually this workflow is not needed, we already have https://github.com/adobe/create-aio-lib/blob/master/.github/workflows/node.js.yml